packages/design-system/src/components/web-components/ds-step-list/ds-step-list.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import type { Meta } from '@storybook/react';
+import WebComponentDocTemplate from '../../../../../../.storybook/docs/WebComponentDocTemplate.mdx';
+import { webComponentDecorator } from '../storybook';
+import { stepListStepData } from '../../StepList/StepList.stories';
+import './ds-step-list';
+
+const meta: Meta = {
+  title: 'Web Components/ds-step-list',
+  decorators: [webComponentDecorator],
+  parameters: {
+    docs: {
+      page: WebComponentDocTemplate,
+      description: {
+        component: `For information about how and when to use this component, [refer to its full documentation page](https://design.cms.gov/components/step-list/).
+
+**Note:** The \`ds-step-list\` is based on the existing functionality of our React component \`StepList\`. Unlike \`StepList\`, which uses an optional \`component\` prop to render custom components for link elements—defaulting to an \`<a>\` tag if not provided—the \`ds-step-list\` does not support the \`component\` attribute. Instead, all link elements in \`ds-step-list\` will default to standard anchor (\`<a>\`) tags to simplify the API and ensure consistent behavior.`,
+      },
+    },
+  },
+  args: {
+    steps: [],
+  },
+  argTypes: {
+    steps: {
+      description: `
+An array of \`StepObjects\` that contain text, state, link URLs, and other info needed to render steps. For more details, refer to the [StepList documentation on storybook](https://design.cms.gov/storybook/?path=/docs/components-steplist--docs). Note: The \`component\` prop is excluded in \`ds-step-list\`.
+`,
+      control: 'text',
+      table: {
+        type: { summary: 'array' },
+      },
+    },
+    'show-sub-sub-steps': {
```

Comment:
We have this option in the story, it would be nice to have it do something in the story. Can you add sub sub steps to the demo data in the Story so this toggle changes something?